pkgs/data/misc/nixos-artwork/wallpapers.nix

```diff
@@ -3,12 +3,18 @@
 let
   mkNixBackground = { name, src, description }:
 
-    stdenv.mkDerivation {
+  let
+    pkg = stdenv.mkDerivation {
       inherit name src;
 
       dontUnpack = true;
 
       installPhase = ''
+        # GNOME
+        mkdir -p $out/share/backgrounds/nixos
+        ln -s $src $out/share/backgrounds/nixos/${src.name}
+
+        # TODO: is this path still needed?
         mkdir -p $out/share/artwork/gnome
```

Comment:
Do not see `artwork` used within path when searching g-c-c. Googling "/usr/share/artwork/gnome" shows no results, without `gnome` only few mentions but none of them look official. We might want to keep this for backwards compatibility, though.